lib/remap-functions/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+mod base64_util;
```

Comment:
:+1: This is definitely better, though I still think I would have preferred to see the functions themselves moved into a submodule.

Something like:

```
lib/remap-functions/src/base64/mod.rs // put charset here
lib/remap-functions/src/base64/encode.rs // put encode function here
lib/remap-functions/src/base64/decode.rs // put decode function here
```

Then you'd import it here like:

```
#[cfg(any(feature = "decode_base64", feature = "encode_base64"))]
mod base64;

#[cfg(feature = "enode_base64")]
pub use  base64::Encode;

#[cfg(feature = "decode_base64")]
pub use  base64::Decode;
```

I see we actually already have a `util` module. I'd also be fine throwing the charset in there for now until it becomes too bloated (naming it `Base64Charset`). My preference is still submodule though.

Reply:
Ah, I see what you mean. As long as @JeanMertz and @FungusHumungus are okay with un-flattening the function dir, I support this.

Reply:
Okay, I've moved everything into a separate module but I've named it `base64_encoding` to avoid some opposite-of-fun naming collisions with the `base64` dependent. Open to suggestion on better names there.

Reply:
I actually prefer the flat-file hierarchy, as it makes getting a list of functions (and iterating over them in an editor) easier.

We already have a `utils.rs` in place with `cfg`'s to only include what's needed for specific functions.

I'll update this once the new parser has landed 👍

Reply:
:+1: I'm cool with that too.

Reply:
@JeanMertz Don't sweat it, I opened a quick PR for that (#6335)